scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -53,10 +52,31 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
     };
     this.activeFilterTags = {};
 
+    Promise.all(
+        Object.values(extensions)
+            .map(
+                (extension) =>
+                    extension.activationResult?.contributions?.personalSpace?.getSections?.(),
+            ),
+    ).then((res) => {
+        const response = _.flattenDeep(res);
+
+        response.forEach((resp) => {
+            if (resp?.type === 'markedForMe') {
```

Comment:
If you use a default that I suggested in the other comment, `resp` will never be null, thus you don't need this condition. Also it's not good that specific strings from an extension are used in the core. The purpose of extensions is decoupling. The code in the core should not make any assumptions about what is in extension code. It should only implement the API.